testing/py3-commonmark/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-commonmark"
+_pyname="commonmark"
+pkgver="0.9.0"
+pkgrel=0
+pkgdesc="Python parser for the CommonMark Markdown specification"
+url="https://pypi.org/project/commonmark/"
+arch="noarch"
+license="BSD-3-Clause"
+depends="python3 py3-docutils py3-future"
+makedepends="py3-setuptools py3-hypothesis"
```

Comment:
Seems like py3-hypothesis should be in `checkdepends` (and maybe py3-flake8 too) https://github.com/readthedocs/commonmark.py/blob/5711cc5b535559af6f88b41ac8d409662fb70085/setup.py#L23

Reply:
Fixed